packages/meteor-gazelle-ipmanager/lib/loginAttempts.js

```diff
@@ -0,0 +1,56 @@
+LoginAttempts = new Mongo.Collection('loginAttempts');
+LoginAttempt = Astro.Class({
+  name: 'LoginAttempt',
+  collection: LoginAttempts,
+  fields: {
+    ip: {
+      type: 'number',
+      index: 1
```

Comment:
Didn't know you can add indexes when defining fields, confirm this is true and if so need to decide on standards